pybossa/syncer/project_syncer.py

```diff
@@ -81,6 +81,17 @@ def _build_payload(self, project, target=None):
 
         if not target:
             payload = self._remove_reserved_keys(project_dict)
+            input_data = output_data = ""
+            if "data_classification" in project.info and "input_data" in project.info["data_classification"]:
+                input_data = project.info["data_classification"]["input_data"]
+            if "data_classification" in project.info and "output_data" in project.info["data_classification"]:
+                output_data = project.info["data_classification"]["output_data"]
+            data_class = str(input_data) + str(output_data)
+            # using .get in case annotation_config doesn't exist 
+            annotation_config = payload['info'].get('annotation_config', {})
+            if ('L1' in data_class or 'L2' in data_class):
+                annotation_config['amp_store'] = False
+            payload['info']['annotation_config'] = annotation_config
```

Comment:
For a L1 project with ```amp_store = True```, line 93 will sync with False value and line 94 will use the original reference, which is True. Is it desired?